Source/RapyutaSimulationPlugins/Private/Tools/ROS2Spawnable.cpp

```diff
@@ -8,7 +8,7 @@
 void UROS2Spawnable::InitializeParameters(const FROSSpawnEntity_Request& InRequest)
 {
     SetName(InRequest.state_name);
-    SetNamespace(InRequest.robot_namespace);
+    SetNamespace(InRequest.robot_namespace.Replace(TEXT("/"),TEXT("")));
```

Comment:
It looks like we would not want any `/` at all in `robot_namespace`, not just as a leading character. If so, the PR description could have a bit of clarifying update I guess.

Reply:
It looks like we would not want any `/` at all in `robot_namespace`, not just as a leading character. If so, the PR description could have a bit of clarifying update I guess.
Also, maybe this could use a log message to let users know about the trimming?